### PR TITLE
Maintain | Exclude tests, specs & behat from zip archives

### PR DESCRIPTION
**Pull request by stloyd**
| Q               | A
| --------------- | -----
| Branch?         | 1.10
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no*
| Deprecations?   | no
| License         | MIT

Please read for Symfony decision why this is good idea: https://github.com/symfony/symfony/pull/33579

This also improves our 🌍 impact as developers cause smaller zips, mean less traffic, which reduces can a little reduce amount of pollution 🏭  !

Current `master` | My PR
------------ | -------------
Zip ~32M | Zip ~18M

### CODE REVIEW

**Comment by oallain** [comment]
Hello,

Thanks for that ❤️ 
Have you been able to do a benchmark to see the benefit of this change?

**Comment by oallain** [comment]
> @oallain
> 
> Current `master`	My PR
> Zip ~32M	Zip ~18M
> So I guess that reducing size by almost half is quite good result

Yes, this is a good result 💪🏼

**Review by oallain — APPROVED** [review]

File: src/Sylius/Bundle/AddressingBundle/.gitattributes [hunk 1]
```diff
@@ -0,0 +1,9 @@
+/Behat export-ignore
```
Comment (pamil) [inline src/Sylius/Bundle/AddressingBundle/.gitattributes:1]: I think this directory exists only in ApiBundle.

File: src/Sylius/Bundle/AddressingBundle/.gitattributes [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+/fixtures export-ignore
```
Comment (pamil) [inline src/Sylius/Bundle/AddressingBundle/.gitattributes:2]: Not sure about this one, but I'm not sure if any bundle has this directory.

File: .gitattributes [hunk 3]
```diff
@@ -0,0 +1,14 @@
+/adr export-ignore
+/docs export-ignore
+/features export-ignore
+/phpspec.yml.dist export-ignore
+/phpstan.neon.dist export-ignore
+/phpunit.xml.dist export-ignore
+/psalm.xml export-ignore
+/src/Sylius/Behat export-ignore
```
[the remark this answers is not in the document]
Comment (victor-v-bv) [reply .gitattributes:8]: Just run composer install with `--prefer-source`.
Perhaps a note somewhere will help.

**Pull request closed without merging**